pkg/controller/machineautoscaler/machinetarget.go

```diff
@@ -186,3 +186,31 @@ func (mt *MachineTarget) GetOwner() (types.NamespacedName, error) {
 
 	return nn, nil
 }
+
+// RemoveAnnotations removes the annotations with the given keys from the
+// target.  It returns a bool indicating whether the annotations were actually
```

Comment:
Just curious about how the caller distinguishes between found/not found when multiple annotations are passed but only some are deleted.

Reply:
I'm not really sure this is a great API... We can change it at some point. In any case, this is really just meant to signal that the annotations changed in some way -- nothing more specific than that. It makes it easier to know if you need to update the resource with the API or not.